profiles/serializers_test.py

```diff
@@ -51,10 +61,14 @@ def test_full(self):  # pylint: disable=no-self-use
             'birth_city': profile.birth_city,
             'preferred_language': profile.preferred_language,
             'pretty_printed_student_id': profile.pretty_printed_student_id,
+            'edx_level_of_education': profile.edx_level_of_education,
+            'education': [
+                EducationSerializer().to_representation(education) for education in profile.education.all()
+                ],
```

Comment:
Can you change the indentation here and in `work_history` below to remove four spaces, so that the `]` lines up with `'education'`?